sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -190,21 +190,26 @@ case class CreateViewCommand(
       // added/generated from a temporary view.
       // 2) The temp functions are represented by multiple classes. Most are inaccessible from this
       // package (e.g., HiveGenericUDF).
-      child.collect {
-        // Disallow creating permanent views based on temporary views.
-        case UnresolvedRelation(AsTableIdentifier(ident))
+      def verify(child: LogicalPlan) {
```

Comment:
Why did you wrap this verification code with the function `verify`? I don't think we need  it.

Reply:
because it needs to call it for SubqueryExpression recursively?

Reply:
Ur, my bad, I see. Thanks!